specs/Deployment/MSIXPackageVersioning.md

```diff
@@ -403,6 +403,7 @@ and release 1.x...
 
 * WARfwk: `Microsoft.WindowsAppRuntime.<rmajor>.<rminor>[-tag]`
 * WARmain: `MicrosoftCorporationII.WinAppRuntime.Main.<rmajor>.<rminor>[-shorttag]`
+* WARmain: `MicrosoftCorporationII.WinAppRuntime.Singleton[-shorttag]`
```

Comment:
WARsingleton (not WARmain).

btw, should WAR be changed to MSIX or WAS, here too ?

Reply:
Fixed naming WARmain to WARsingleton for all singleton package names in the file